src/cmd/format.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"gopkg.in/yaml.v3"
+)
+
+var outputFormat string
+
+type encoder interface {
+	Encode(data any) error
+}
+
+func init() {
+	rootCmd.PersistentFlags().StringVarP(&outputFormat, "output", "o", "json", "output format")
+}
+
+func getEncoder() encoder {
+	switch outputFormat {
+	case "yaml":
+		enc := yaml.NewEncoder(os.Stdout)
+		enc.SetIndent(2)
+
+		return enc
```

Comment:
I'm wondering if yaml is worth bring into the cli. It's an extra dependency that might not be needed? If people need yaml, `yq` can do the job and will compose well: `| yq ...etc...`